### PR TITLE
German Translation updated to 2022 version

### PR DESCRIPTION
**Pull request by LauraHolzhauer**
German translations updated to the 2022 regulations+guidelines

Co-Authored-By: Sébastien Auroux <3089192+Sauroux@users.noreply.github.com>
Co-Authored-By: Laura Ohrndorf <362249+Laura-O@users.noreply.github.com>

### CODE REVIEW

**Comment by hanwu85** [comment]
Hi @LauraHolzhauer ,

Thank you for the contribution and the  pull request.
There are some trailing white spaces, could you please fix them according to the details of the checks?

**Comment by LauraHolzhauer** [comment]
Hi @hanwu85,

sorry for the inconvenience, those errors should be fixed now.

File: german/wca-guidelines.md [hunk 1]
```diff
@@ -143,31 +136,38 @@ Jede Richtlinie ist einer der folgenden Kategorien zugeordnet. Diese Beschriftun
...
+
+
 ## <article-A><speedsolving><speedsolving> Abschnitt A: Speed Solving (Lösen auf Zeit)
 
 - A1a2+) [Zusatz] Im Falle eines kumulativen Zeitlimits schreibt der Judge die für einen DNF gestoppte Zeit in Klammer auf das Scoresheet, z.B. "DNF (1:02.27)".
-- A1a2++) [Erklärung] Kumulative Zeitlimits dürfen für eine Kategorie oder eine Kombination von Runden von unterschiedlichen Kategorien gesetzt werden.
+A1a2++) [Beispiel] Wenn bei einer Runde mit einem kumulativen Zeitlimit von 30 Minuten die ersten beiden Ergebnisse eines Teilnehmers 6:00 und DNF (10:00) sind, ist das Zeitlimit für den dritten Versuch 30:00 - 6:00 = 14:00. In diesem Fall ist zu beachten, dass nur das endgültige Ergebnis für einen Versuch zählt. Hierbei würde ein dritter Versuch mit einer gestoppten Zeit von 13:59 ohne Zeitstrafen für das Zeitlimit ausreichen, da da das finale Ergebnis 13:59 ist (siehe [Regel 9f2](regulations:regulation:9f2)), während ein dritter Versuch mit einem Ergebnis von 13:59 + 2 = 14:01 das Zeitlimit nicht erfüllen würde.
```
Comment (hanwu85) [inline german/wca-guidelines.md:153]:
```suggestion
- A1a2++) [Beispiel] Wenn bei einer Runde mit einem kumulativen Zeitlimit von 30 Minuten die ersten beiden Ergebnisse eines Teilnehmers 6:00 und DNF (10:00) sind, ist das Zeitlimit für den dritten Versuch 30:00 - 6:00 = 14:00. In diesem Fall ist zu beachten, dass nur das endgültige Ergebnis für einen Versuch zählt. Hierbei würde ein dritter Versuch mit einer gestoppten Zeit von 13:59 ohne Zeitstrafen für das Zeitlimit ausreichen, da da das finale Ergebnis 13:59 ist (siehe [Regel 9f2](regulations:regulation:9f2)), während ein dritter Versuch mit einem Ergebnis von 13:59 + 2 = 14:01 das Zeitlimit nicht erfüllen würde.
```

File: german/wca-regulations.md [hunk 2]
```diff
@@ -201,41 +202,39 @@ Anmerkung: Da die Nummerierung der Abschnitte und Regeln beim Löschen von Regel
...
 		- 7f1b) Matte: Die Matte einer Stackmat.
 		- 7f1c) Timer: Der Timer einer Stackmat oder eine Stoppuhr (bei längeren Versuchen).
 		- 7f1d) Oberfläche: Die flache Oberfläche, auf der die Stackmat platziert wurde. Die Matte wird als Teil der Oberfläche angesehen, der Timer jedoch nicht.
 	- 7f2) Der Timer muss an der Matte angebracht sein und auf der Oberfläche platziert sein, so dass die Seite mit dem Timer dem Teilnehmer am nächsten ist.
-		- 7f2a) Ausnahme: Für das Lösen mit Füßen wird die Stackmat direkt auf dem Boden platziert. Der Timer darf dann außerdem so ausgerichtet werden, dass die Seite mit dem Timer am weitesten vom Teilnehmer entfernt ist.
-- 7h) Der Veranstaltungsort muss einen Wartebereich für die Teilnehmer haben.
+	- 7f3) Das Organisationsteam sollte Stackmat Timer dahingehend modifizieren, dass sie widerstandsfähiger gegen geläufige Zwischenfälle sind. Dazu gehört, dass die Knöpfe schwieriger versehentlich zu drücken sind (z.B. durch das Anbringen von O-Ringen um die Knöpfe) und die Batterie stärker fixiert wird (z.B. durch Polsterung des Batteriefachs).
+	7f4) Generation 5 Pro Timer müssen im 2-Pad-Modus verwendet werden.
```
Comment (hanwu85) [inline german/wca-regulations.md:211]:
```suggestion
	- 7f4) Generation 5 Pro Timer müssen im 2-Pad-Modus verwendet werden.
```

File: german/wca-guidelines.md [hunk 3]
```diff
@@ -143,31 +136,38 @@ Jede Richtlinie ist einer der folgenden Kategorien zugeordnet. Diese Beschriftun
...
-- 11e2+) [Zusatz] Wenn ein Extraversuch selbst zu einem weiteren Extraversuch führt, sollte der Teilnehmer solange Versuche machen, bis genug vorhanden sind, um so viele reguläre Ergebnisse zu ersetzen wie benötigt. Die gültigen Extraversuche ersetzen die relevanten regulären Versuche in aufsteigender Reihenfolge.
+- 11e2+) [Zusatz] Wenn ein Extraversuch selbst zu einem weiteren Extraversuch führt, sollte der Teilnehmer solange Versuche machen, bis genug vorhanden sind, um so viele reguläre Ergebnisse zu ersetzen wie benötigt.
 - 11e2++) [Beispiel] Angenommen ein Teilnehmer hat 5 reguläre Versuche und der 2. Versuch führt zu einem Extraversuch. Der Teilnehmer sollte nun Extraversuche durchführen, bis ein gültiger Versuch vorhanden ist, um den 2. Versuch zu ersetzen. Wenn spätere Versuche zu weiteren Extraversuchen führen, sollten diese in gleicher Weise durchgeführt werden.
 
 
+## <article-11><incidents><incidents> Abschnitt 11: Zwischenfälle
```
Comment (hanwu85) [inline german/wca-guidelines.md:145]: Title 11 is placed in title 12's line.  Please fix this one.

File: german/wca-regulations.md [hunk 4]
```diff
@@ -510,16 +520,32 @@ Anmerkung: Da die Nummerierung der Abschnitte und Regeln beim Löschen von Regel
...
+- Y6) Umgebung:
+	- Y6a) Wenn der WCA-Delegierte dies als nötig erachtet, muss der Teilnehmerbereich keinen Wartebereich für die Teilnehmer aufweisen.
+	- Y6b) Teilnehmer sollten nicht die Oberfläche der Lösestation berühren, wenn sie gerade keinen offiziellen Versuch durchführen.
+- Y7) Teilnahmeabläufe, die die Artikel [A](regulations:article:A), [B](regulations:article:B) und [H](regulations:article:H) ersetzen, sind im Folgenden beschrieben. Diese Abläufe gelten nicht für Artikel [E](regulations:article:E), Lösen mit möglichst wenigen Zügen.
+	- Y7a) Wettbewerbe sollten das Fixed-Seating-Konzept benutzen (siehe [Regel Y2b](regulations:regulation:Y2b)).
+	- Y7b) Der Judge sollte versuchen, das Puzzle des Teilnehmers möglichst nicht zu berühren. Ausnahme: Für Clock sollte der Judge das Puzzle aufheben, um zu überprüfen, dass beide Seiten gelöst sind.
+	- Y7c) Scrambling:
+		- Y7c1) Wenn der Teilnehmer zu einer Runde aufgerufen wird, gibt er sein Puzzle in desinfiziertem und gelöstem Zustand beim Scrambler ab. Der Teilnehmer wartet dann an der ihm zugewiesenen Lösestation.
+	- Y7d) Inspektion:
```
Comment (hanwu85) [inline german/wca-regulations.md:547]:
```suggestion
	- Y7d) Inspektion:
```

**Comment by hanwu85** [comment]
Hi @LauraHolzhauer 
I have added some suggestions which I think will solve the problems.
Please check them, thanks!

File: german/wca-regulations.md [hunk 5]
```diff
@@ -350,31 +356,31 @@ Anmerkung: Da die Nummerierung der Abschnitte und Regeln beim Löschen von Regel
...
-		- A2d1) Das Organisationsteam darf Scrambler auffordern zusätzliche Maßnahmen zu ergreifen, um sicherzustellen, dass sie den korrekten Scramble angewandt haben (z.B. Unterschreiben des Scoresheets um zu bestätigen, dass sie den richtig nummerierten Scramble angewandt haben und dass dieser dem Bild entspricht, oder durch die Nutzung zusätzlicher Personen als "Scramble Checker").
-	- A2e) Der Judge legt das Puzzle in zufälliger Orientierung auf der Stackmat ab und stellt sicher, dass es komplett verdeckt bleibt.
+		- A2d1) Der Scrambler oder ein autorisierter Judge müssen das gescrambelte Puzzle überprüfen und auf dem Scoresheet unterschreiben/ihre Initialen setzen, dass sie den korrekt nummerierten Scramble angewandt haben und überprüft haben, dass dieser mit dem Bild übereinstimmt (siehe [Regel 4g](regulations:regulation:4g)). Ausnahme: für den 6x6x6 Würfel, 7x7x7 Würfel und Megaminx darf der Scrambler unterschreiben/seine Initialen setzen, dass das Puzzle hinreichend gescrambelt ist (siehe [Regel 4g1](regulations:regulation:4g1)).
+	- A2e) Der Judge legt das Puzzle in zufälliger Orientierung auf der Stackmat ab und stellt sicher, dass es komplett verdeckt bleibt.
```
Comment (hanwu85) [inline german/wca-regulations.md:378]:
```suggestion
	- A2e) Der Judge legt das Puzzle in zufälliger Orientierung auf der Stackmat ab und stellt sicher, dass es komplett verdeckt bleibt.
```

File: german/wca-regulations.md [hunk 6]
```diff
@@ -510,16 +521,32 @@ Anmerkung: Da die Nummerierung der Abschnitte und Regeln beim Löschen von Regel
...
-	- Z4c) Falls es mehr als eine Registrierungsphase gibt, muss das Organisationsteam diese jeweils mindestens 48 Stunden vor dem Start der zusätzlichen Registrierungsphase ankündigen.
-- Z5) Das Organisationsteam darf Teilnehmern verbieten, in bestimmten Kombinationen von Kategorien teilzunehmen.
+	- H1e) Der Teilnehmer darf keine Züge an einem oder mehreren Puzzles während der Memorisierungsphase durchführen. Strafe: Disqualifikation des Versuchs (DNF). Ausnahme: diese einzelnen Puzzles dürfen als ungelöste Puzzles am Ende des Versuchs eingerechnet werden, anstatt den gesamten Versuch zu disqualifizieren; dies geschieht nach Ermessen des WCA-Delegierten.
+- H2) Der Teilnehmer darf selbst eine flache Unterlage für die Oberfläche (z.B. eine Matte oder ein Tuch) mitbringen, nach Ermessen des WCA-Delegierten. Wenn der Teilnehmer dies tun möchte, muss der Wunsch an dem Zeitpunkt geäußert werden, wenn die Anzahl der Puzzles dem Organisationsteam mitgeteilt wird (siehe [Regel H1a](regulations:regulation:H1a)).
+
+## <article-Y><temporary regulations><temporary regulations> Abschnitt Y: Temporäre Regeln
```
Comment (hanwu85) [inline german/wca-regulations.md:527]:
```suggestion
## <article-Y><temporary-regulations><temporaryregulations> Abschnitt Y: Temporäre Regeln
```

**Comment by hanwu85** [comment]
Hi @LauraHolzhauer !
I think all error has been fixed.  Please check the last 2 commit done by me.
If you think they are okay.  I will merge this PR.

Thanks!

**Comment by LauraHolzhauer** [comment]
Hi @hanwu85,

everything seems fine to me. Thank you as well!

**Review by Ivan-Brigidano — APPROVED** [review]
LGTM!
@hanwu85 do you mind making the PR of the translations index?

**Comment by hanwu85** [comment]
@LauraHolzhauer  
Thank you very much for the this translation! 

@Ivan-Brigidano 
Yes, I will update the index page.

**Pull request merged**